lib/LoRa_APRS/LoRa_APRS.h

```diff
@@ -19,13 +19,16 @@ class LoRa_APRS : public LoRaClass {
   void setRxFrequency(long frequency);
   long getRxFrequency() const;
 
+  void setRxGain(uint8_t gain);
+
   void setTxFrequency(long frequency);
   long getTxFrequency() const;
 
 private:
   std::shared_ptr<APRSMessage> _LastReceivedMsg;
   long                         _RxFrequency;
   long                         _TxFrequency;
+  uint8_t                      _RxGain;
```

Comment:
there is no need to have the gain in this class.
please set the gain in the `setRxGain()` function and not save it in the class. this variable is not even read somewhere.

Reply:
Fixed this. And set the default for gain_rx to "0", i.e. AGC, in project_configuration.cpp

Reply:
please fix the value also in the json file

Reply:
Done